README.md

```diff
@@ -332,24 +326,21 @@ bar.default(); // 2 ( assigned default function )
 bar(); // throws, bar is not a function
 ```
 
-## CJS => ES Modules
+### Examples: Consuming ES Modules with CommonJS
 
 <!--
 反対に CommonJS から ES Modules を読み込む時は下記のようになります。こちらは `export default` で export した場合は `.default` プロパティにアサインされます。
 -->
 
-Next samples are opposite direction.
-
 <!--
 export default を利用する例:
 -->
 
-Sample 1, using export default:
+**Example 1: Using `export default`**
 
 ```javascript
 // es.mjs
 let foo = {bar:'my-default'};
-// note:
 export default foo;
 foo = null; // this null value does not effect import value.
```

Comment:
`export default foo` ~= `export let default = foo`

It is hard to represent the "live binding" of things using `default`.

Having a 2nd export might help explain that live bindings still work.

```js
// export the *variable* foo
export let foo = {bar:'my-default'};

// note: this exports the current *value* of foo
export default foo;

// this null value does not affect the default export
foo = null; 
```